src/test/common/fdc3.app-channels.ts

```diff
@@ -192,7 +196,7 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
 
         const testChannel = await cc.createTestChannel()
         const resolveExecutionCompleteListener = cc.initCompleteListener(acTestId11)
-        cc.openChannelApp(acTestId11, undefined, APP_CHANNEL_AND_BROADCAST_TWICE)
+        await cc.openChannelApp(acTestId11, testChannel.id, APP_CHANNEL_AND_BROADCAST_TWICE)
         await resolveExecutionCompleteListener;
 
         const context = await testChannel.getCurrentContext();
```

Comment:
These test should be extended to also check that adding a context listener does not auto-replay the current context.